addons/medical_engine/script_macros_config.hpp

```diff
@@ -24,27 +26,21 @@
 // the hit point itself should not take any damage
 // It is important that the "ACE_HDBracket" hit point is the last in the config,
 // but has the same selection as the first one (always "HitHead" for soldiers).
-#define ADD_ACE_HITPOINTS(arm_armor,leg_armor)\
-    class HitLeftArm {\
-        armor = arm_armor;\
+#define ADD_ACE_HITPOINTS\
+    class HitLeftArm: HitHands {\
         material = -1;\
         name = "hand_l";\
-        passThrough = 1;\
         radius = 0.08;\
-        explosionShielding = 1;\
         visual = "injury_hands";\
         minimalHit = 0.01;\
     };\
     class HitRightArm: HitLeftArm {\
         name = "hand_r";\
     };\
-    class HitLeftLeg {\
-        armor = leg_armor;\
+    class HitLeftLeg: HitLegs {\
         material = -1;\
         name = "leg_l";\
-        passThrough = 1;\
         radius = 0.1;\
-        explosionShielding = 1;\
         visual = "injury_legs";\
         minimalHit = 0.01;\
     };\
```

Comment:
Previously the values used here were set manually and often didn't match up to the vanilla hitpoints. In the interest of vanilla-consistency I think it makes more sense to just inherit them.

Reply:
Good idea to include a comment above the macro to explain their absence just in case anyone thinks of adding them again in future (also for reviewers who may not see this)

Reply:
It's impressive that you're willing to write a multiline explanation into the code using single-line comments, due to how laborious it is, I'll give you that, but I still think it's better to use the `/* ... */` instead XD

Unless it's a policy I missed.